code/drasil-website/lib/Drasil/Website/Body.hs

```diff
@@ -14,6 +14,17 @@ import Drasil.Website.Example (exampleSec, exampleRefs, allExampleSI)
 import Drasil.Website.Documentation (docsSec, docRefs)
 import Drasil.Website.Analysis (analysisSec, analysisRefs)
 import Drasil.Website.GettingStarted (gettingStartedSec)
+import Data.Drasil.Concepts.Physics (pendulum, motion, rigidBody)
```

Comment:
Something is very wrong here: there is no way the website should depend on all these things added here!

Reply:
The reason the website depends on all these things added here is that we are now using the purposes as defined in `Body.hs` of each of the examples. Since those purposes make use of these terms when writing the `Sentence`, the website now has to depend on these imports.